src/coprocessor/dag/expr/ctx.rs

```diff
@@ -273,27 +329,26 @@ mod test {
     #[test]
     fn test_handle_truncate() {
         // ignore_truncate = false, truncate_as_warning = false
-        let mut ctx = EvalContext::new(Arc::new(EvalConfig::new(0).unwrap()));
+        let mut ctx = EvalContext::new(Arc::new(EvalConfig::new()));
         assert!(ctx.handle_truncate(false).is_ok());
         assert!(ctx.handle_truncate(true).is_err());
         assert!(ctx.take_warnings().warnings.is_empty());
         // ignore_truncate = false;
-        let mut ctx = EvalContext::new(Arc::new(EvalConfig::new(FLAG_IGNORE_TRUNCATE).unwrap()));
+        let mut ctx = EvalContext::new(Arc::new(EvalConfig::mock()));
```

Comment:
This is just reusing same code, since this code has been repeatedly-written for many, many, many times and each time you guys ask me to refactor the usage I have those many, many, many, many use cases to change. If you mean this line specifically, it's reasonable to me.